### PR TITLE
Document how to use this container for exposing host ports

### PR DESCRIPTION
**Pull request by StefanLobbenmeierObjego**
I heard about this feature at Devoxx Belgium 2024 and wanted to give this a try, but could not find the documentation for it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
 # sshd-docker
 Docker image with SSH daemon installed
+
+# Usage
+This container is meant to be used to expose host ports to containers running in Docker.
+When using testcontainers in unit tests, refer to the documentation of testcontainers to find out how to use it:
+- Java: https://java.testcontainers.org/features/networking/#exposing-host-ports-to-the-container
+- .NET: https://dotnet.testcontainers.org/api/create_docker_network/#exposing-host-ports-to-the-container
+- node.js: https://node.testcontainers.org/features/networking/#expose-host-ports-to-container
```
Comment (mdelapenya) [inline README.md:9]: Could you add Go too?
```suggestion
- node.js: https://node.testcontainers.org/features/networking/#expose-host-ports-to-container
- Go: https://golang.testcontainers.org/features/networking/#exposing-host-ports-to-the-container
```

Comment (StefanLobbenmeierObjego) [reply README.md:9]: sounds good, I also tried finding it in haskell and rust but did not see it in the docs

File: README.md [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+- Go: https://golang.testcontainers.org/features/networking/#exposing-host-ports-to-the-container
+
+Alternatively, you can start the container yourself:
+```yaml
```
Comment (mdelapenya) [inline README.md:13]: This LGTM, although I'm trying to reproduce this scenario with compose and the python http server locally without success. Can you double check it works 🙏 ?

Once verified, we can merge this, thanks!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,40 @@
...
+    command: /bin/sh -c "while true; do curl http://sshd:8080; sleep 5; done"
+    depends_on:
+      - sshd
+```
+
+And connect to it via ssh:
+
+```bash
+# ssh -R [remote_port]:[destination_address]:[local_port] [username]@[ssh_server]
+ssh -R 8080:localhost:8080 root@localhost:10022
```
Comment (StefanLobbenmeierObjego) [inline README.md]:
```suggestion
# ssh -R [remote_port]:[destination_address]:[local_port] [username]@[ssh_server] -p [ssh_port]
ssh -R 8080:localhost:8080 root@localhost -p 10022
```

Good catch, this wrong the wrong syntax for the port

File: README.md (hunk 2 above)
Comment (StefanLobbenmeierObjego) [reply README.md:13]: so after the port fix this is how it looks like on my machine:
image


You might get a few connection refused at the very start while the server is not running, but once it is started you should see requests on the python server and the responses in the docker logs

Comment (StefanLobbenmeierObjego) [reply README.md:13]: I also published the folder here: https://github.com/StefanLobbenmeierObjego/testcontainers-sshd-example but should be the same content as here

Comment (mdelapenya) [reply README.md:13]: Cool, thanks for sharing! I cloned the repro project and it worked! I'm sure I did something wrong with my previous setup, so than you for taking the time to explaining it to me.

Yeah, this LGTM!

**Review by mdelapenya — APPROVED** [review]
LGTM, thanks!

**Pull request merged**